api/yelp_beans/logic/user.py

```diff
@@ -203,11 +203,12 @@ def add_preferences(user, updated_preferences, subscription_id):
     set(SubscriptionDateTime.id)
     """
     added = set()
-    for datetime_id, active in updated_preferences.items():
-        if active:
+    for datetime_id, status in updated_preferences.items():
+        if status.enabled:
```

Comment:
instead of passing a boolean to the backend, it's a dictionary now that looks something like `{enabled: true, auto_renew: false}`